test/verify/check-connection

```diff
@@ -88,6 +88,16 @@ class TestConnection(MachineCase):
         b.expect_load()
         b.enter_page("/system")
 
+        # Lets crash a process and see if we get a proper backtrace in the logs
+        # This helps with debugging failures in the tests elsewhere
+        if m.image in ["debian-unstable"]:
+            cron = "cron"
+        else:
+            cron = "crond"
+        m.execute("systemctl start {0}".format(cron))
```

Comment:
I'm not entirely sure this meta-test (test to check if our test setup works) needs to be in `check-connection`.